datacommons/examples/core.py

```diff
@@ -49,41 +44,13 @@ def main():
       print('  - {}'.format(city_dcid))
 
   # Print the first 10 triples associated with Santa Clara County
-  utils._print_header('Triples for Santa Clara County')
+  print('Triples for Santa Clara County')
   triples = dc.get_triples(dcids)
   for dcid in dcids:
     print('> Triples for {}'.format(dcid))
     for s, p, o in triples[dcid][:5]:
       print('  - ("{}", {}, "{}")'.format(s, p, o))
 
-  # get_property_values can be easily used to populate Pandas DataFrames. First
-  # create a DataFrame with some data.
-  utils._print_header('Initialize the DataFrame')
-  pd_frame = pd.DataFrame({'county': ['geoId/06085', 'geoId/24031']})
-  print(pd_frame)
-
-  # Get the names for the given counties.
-  utils._print_header('Get County Names')
-  pd_frame['county_name'] = pd_frame['county'].map(
-    dc.get_property_values(pd_frame['county'], 'name'))
-  pd_frame = pd_frame.explode('county_name')
-  print(pd_frame)
-
-  # Get the cities contained in these counties.
-  utils._print_header('Get Contained Cities')
-  pd_frame['city'] = pd_frame['county'].map(
-    dc.get_property_values(
-      pd_frame['county'], 'containedInPlace', out=False, value_type='City'))
-  pd_frame = pd_frame.explode('city')
-  print(pd_frame)
-
-  # Get the names for each city.
-  utils._print_header('Get City Names')
-  pd_frame['city_name'] = pd_frame['city'].map(
-    dc.get_property_values(pd_frame['city'], 'name'))
-  pd_frame = pd_frame.explode('city_name')
-  print(pd_frame)
```

Comment:
Do we not want to demo these API calls any more?

Reply:
pandas is creating issues when using with Bazel, it can make the entire example not running, and it is the reason to remove them.  Also these are supplemental but not critical for the example, as the API itself does not mention pandas at all.